distributions/aws/examples/storage-efs/README.md

```diff
@@ -0,0 +1,153 @@
+
+## 5.3 Using EFS volume as workspace or data volume for a notebook server 
+
+Spin up a new Kubeflow notebook server and specify the name of the PVC to be used as the workspace volume or the data volume and specify your desired mount point. For our example here, we are using the `AWS Optimized Tensorflow 2.6 CPU image` provided in the notebook configuration options. Additionally, use the existing `efs-claim` volume as the workspace volume at the default mount point `/home/jovyan`. The server might take a few minutes to come up. 
+
+In case the server does not start up in the expected time, do make sure to check - 
+1. The Notebook Controller Logs
+2. The specific notebook server instance pod's logs
+
+### Note about Permissions
+You might need to specify some additional directory permissions on your worker node before you can use these as mount points. By default, new Amazon EFS file systems are owned by root:root, and only the root user (UID 0) has read-write-execute permissions. If your containers are not running as root, you must change the Amazon EFS file system permissions to allow other users to modify the file system. The set-permission-job.yaml is an example of how you could set these permissions to be able to use the efs as your workspace in your kubeflow notebook. 
+```
+kubectl apply -f storage-efs/sample/set-permission-job.yaml
+```
+If you use EFS for other purposes (e.g. sharing data across pipelines), you don’t need this step.
+
+## 5.4 Using EFS volume for a TrainingJob using TFJob Operator
+The following section re-uses the PVC and the Tensorflow Kubeflow Notebook created in the previous steps to download a dataset to the EFS Volume. Then we spin up a TFjob which runs a image classification job using the data from the shared volume. 
+Source: https://www.tensorflow.org/tutorials/load_data/images
+
+### 1. Download the dataset to the EFS Volume 
+In the Kubeflow Notebook created above, use the following snippet to download the data into the `/home/jovyan/.keras` directory (which is mounted onto the EFS Volume). 
+```
+import pathlib
+import tensorflow as tf
+dataset_url = "https://storage.googleapis.com/download.tensorflow.org/example_images/flower_photos.tgz"
+data_dir = tf.keras.utils.get_file(origin=dataset_url,
+                                   fname='flower_photos',
+                                   untar=True)
+data_dir = pathlib.Path(data_dir)
+```
+
+### 2. Build and Push the Docker image
+In the `training-sample` directory, we have provided a sample training script and Dockerfile which you can use as follows to build a docker image- 
+```
+export dockerImage=image-classification:no-data
```

Comment:
nit: export vars are caps in other readmes

Reply:
Did you mean callout to use their own image if they wanted to ?

Reply:
No I mean they will have to change the URI if they have to push to their repo

Reply:
Makes sense, I actually removed the hardcoded defaults altogether, they didnt add any value.